softwareconfiguration/model/build.gradle

```diff
@@ -0,0 +1,61 @@
+dependencies {
+	compile project(':JAICore:jaicore-basic')
+	
+	testCompile project(path: ':JAICore:jaicore-basic', configuration: 'testArtifacts')
+}
+			
+uploadArchives {
+	repositories {
+		mavenDeployer {
+			def ossrhUsername =  project.hasProperty('ossrhUsername') ? project.property('ossrhUsername') : ""
+			def ossrhPassword = project.hasProperty('ossrhPassword') ? project.property('ossrhPassword') : ""
+		
+			beforeDeployment { MavenDeployment deployment -> signing.signPom(deployment) }
+			repository(url: "https://oss.sonatype.org/service/local/staging/deploy/maven2/") {
+        		authentication(userName: ossrhUsername, password: ossrhPassword)
+      		}	
+      		snapshotRepository(url: "https://oss.sonatype.org/content/repositories/snapshots/") {
+        		authentication(userName: ossrhUsername, password: ossrhPassword)
+      		}
+
+      		pom.project {
+        		name 'HASCO'

```

Comment:
here the project name still needs to be adapted. Otherwise, this sub-project will clash with the HASCO project when pushing a new release to mvnrepository.

Reply:
jap